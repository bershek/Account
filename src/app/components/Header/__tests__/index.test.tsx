import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { Header } from '../index';

const shallowRenderer = createRenderer();

describe('<Header />', () => {
  it('should match snapshot', () => {
    shallowRenderer.render(<Header  label="test label" title="test title" onClick={()=> console.log("Test onClick")} />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
