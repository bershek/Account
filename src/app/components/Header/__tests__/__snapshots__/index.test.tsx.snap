// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot 1`] = `
<Header__Wrapper>
  <PageWrapper>
    <Header__Title>
      test title
    </Header__Title>
    <Memo
      label="test label"
      onClick={[Function]}
    />
  </PageWrapper>
</Header__Wrapper>
`;
